ui/raidboss/common_replacement.js

```diff
@@ -28,18 +48,87 @@ let commonReplacement = {
     cn: '狂暴',
     ko: '전멸기',
   },
-  '--untargetable--': {
-    de: '--nich anvisierbar--',
-    fr: '--Impossible à cibler--',
-    cn: '--无法选中--',
-    ko: '--타겟 불가능--',
+  '--jump--': {
+    de: '--sprung--',
+    fr: '--saut--',
+    ja: '--ジャンプ--',
+    cn: '--跳--',
+    ko: '--점프--',
+  },
+  '--middle--': {
+    de: '--mitte--',
+    fr: '--milieu--',
+    cn: '--中间--',
+    ko: '--중간에--',
```

Comment:
I thought this would be better `'--중앙--'`, but there was `'--center--'` upper there, which translates to `'--중앙--'` the same.
what's difference between 'middle' and 'center'? any examples?

Reply:
I think it depends of the shape of the arena. If it is a circular area, then _center_ is more accurate than _middle_.

Reply:
then, this should be `'--중앙--'` rather than `'--중간에--'`.

Reply:
- 중앙 for middle, right ? 
- for --sync, I suspected that it was not useful, but it will prevent it from being rewritten.

Reply:
* 중앙 for middle **AND** center
* I think just removing all the "--sync--" thing written before is better, and tell people to not write it.

Reply:
I didn't know center and middle is differently used in English.
![Screenshot 2020-04-01 at 01 34 33](https://user-images.githubusercontent.com/4695371/78051683-0350ea80-73b9-11ea-80ea-336bc836d0cf.jpg)
중앙 means both center and middle in Korean.

Reply:
There is the same problem for the --reset-- lines, but I cannot integrate it because it exists in (skill-Boss) on O11. This line cannot be deleted and creates a conflict.

I correct for Ko lang.

Reply:
For the rest, I think all the repetitive lines useful or not should be here. So we will be sure not to see them appear elsewhere.